mappings/community/6627839278-registrations.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Can we rename the mocks wit the CRN for consistency with other community mocks?

Reply:
Yes sure, makes sense